Resources/Textures/_Harmony/Structures/Piping/Atmospherics/scrubber.rsi/meta.json

```
@@ -0,0 +1,133 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/pull/32742/commits/44820f5b6e2c864a35bfad448e0a9437f01ad2e2",
```

It's usually easier to test on a merged commit instead of a pull request commit
```suggestion
    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/9b308c274d206fecb0c04165405130255866b8c9",
```